diff -Naur qtwebkit-opensource-src-5.6.0/Source/ThirdParty/gtest/CMakeLists.txt qtwebkit-opensource-src-5.6.0-patched/Source/ThirdParty/gtest/CMakeLists.txt
--- qtwebkit-opensource-src-5.6.0/Source/ThirdParty/gtest/CMakeLists.txt	2016-02-27 07:22:08.000000000 -0500
+++ qtwebkit-opensource-src-5.6.0-patched/Source/ThirdParty/gtest/CMakeLists.txt	2016-04-26 14:34:42.010285736 -0400
@@ -56,7 +56,11 @@
   ${gtest_BINARY_DIR}/src)
 
 # Defines CMAKE_USE_PTHREADS_INIT and CMAKE_THREAD_LIBS_INIT.
+
+
+set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads)
+target_link_libraries(${name} Threads::Threads)
 
 # Defines the compiler/linker flags used to build gtest.  You can
 # tweak these definitions to suit your need.  A variable's value is
