diff -Naur qtwebkit-opensource-src-5.6.0/Source/WTF/wtf/CMakeLists.txt qtwebkit-opensource-src-5.6.0-patched/Source/WTF/wtf/CMakeLists.txt
--- qtwebkit-opensource-src-5.6.0/Source/WTF/wtf/CMakeLists.txt	2016-02-27 07:22:08.000000000 -0500
+++ qtwebkit-opensource-src-5.6.0-patched/Source/WTF/wtf/CMakeLists.txt	2016-04-26 16:57:15.070289815 -0400
@@ -269,6 +269,11 @@
 include_directories(${WTF_INCLUDE_DIRECTORIES})
 add_definitions(-DBUILDING_WTF)
 add_library(WTF STATIC ${WTF_HEADERS} ${WTF_SOURCES})
+
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads)
+target_link_libraries(WTF Threads::Threads)
+
 target_link_libraries(WTF ${WTF_LIBRARIES})
 set_target_properties(WTF PROPERTIES FOLDER "JavaScriptCore")
 
