diff --git a/libzathura/plugin-api.h b/libzathura/plugin-api.h
index daa1503..a8986c8 100644
--- a/libzathura/plugin-api.h
+++ b/libzathura/plugin-api.h
@@ -30,6 +30,12 @@ extern "C" {
 
 /* plugin */
 #define ZATHURA_PLUGIN_REGISTER(plugin_name, major, minor, rev, register_functions, mimetypes) \
+	unsigned int zathura_plugin_version_major(void) { return major; } \
+	unsigned int zathura_plugin_version_minor(void) { return minor; } \
+	unsigned int zathura_plugin_version_revision(void) { return rev; } \
+	unsigned int zathura_plugin_api_version(void) { return ZATHURA_API_VERSION; } \
+	unsigned int zathura_plugin_abi_version(void) { return ZATHURA_ABI_VERSION; } \
+                                                                        \
   const zathura_plugin_version_t zathura_plugin_version = { \
     major, minor, rev \
   }; \
